vmm/sandbox/Cargo.toml

```diff
@@ -7,7 +7,12 @@ edition = "2021"
 [profile.release]
 panic = 'abort'
 
+[build-dependencies]
+built = { version = "0.7.0", features=["cargo-lock", "dependency-tree", "git2", "chrono", "semver"] }
```

Comment:
Is it possiable to remove this dependency as it was already used in `[dependencies]` section? I'm Not sure.

Reply:
Both needed.